src/de/schildbach/pte/VrsProvider.java

```diff
@@ -718,6 +659,9 @@ else if (error.equals("Server Error"))
                 else if (error.equals("Keine Verbindungen gefunden."))
                     return new QueryTripsResult(new ResultHeader(NetworkId.VRS, SERVER_PRODUCT),
                             QueryTripsResult.Status.NO_TRIPS);
+                else if (error.contains("keine") && error.contains("Verbindungen"))
```

Comment:
This match feels too broad to me. Can we not just check for the specific message, like the others?

Reply:
The literal error message was:

>>>
Es wurden keine gültigen Verbindungen für diese Anfrage gefunden.
>>>

I was hesitant because of the u Umlaut in "gültigen", but not what I read through the already implemented error messages, please feel free to replace the if case with the literal message quoted above.

Reply:
I think if there is a problem with Umlauts it would be a good opportunity to check/fix the encodings.